working/templates/srmfunction/ProfileLoadScript_1/ProfileLoadScript_1.csproj

```diff
@@ -11,6 +11,9 @@
     </PropertyGroup>
 
 	<ItemGroup>
+		<PackageReference Include="Newtonsoft.Json">
+		  <Version>13.0.2</Version>
+		</PackageReference>
 		<PackageReference Include="Skyline.DataMiner.Dev.Automation">
```

Comment:
Currently the code does not compile, I suppose the SRM nuget will get added here?

Reply:
OK to wait for SRM NuGet